dev/com.ibm.ws.microprofile.openapi_fat/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -51,8 +51,8 @@ cleanup.use_this_for_non_static_method_access=false
 cleanup.use_this_for_non_static_method_access_only_if_necessary=true
 eclipse.preferences.version=1
 editor_save_participant_org.eclipse.jdt.ui.postsavelistener.cleanup=true
-formatter_profile=_Unmanaged profile
-formatter_settings_version=12
+formatter_profile=_MJS Java
+formatter_settings_version=18
```

Comment:
This is why I personally do not like checking in settings files for the IDEs that are being used to develop project source... but since it is just for formatter settings, I do not believe that it will be an issue.